frontend/tty-receiver/tty-receiver.ts

```diff
@@ -20,6 +20,7 @@ class TTYReceiver {
             scrollback: 0,
             fontSize: 12,
             letterSpacing: 0,
+            fontFamily: 'SourceCodePro'
```

Comment:
It would be awesome if we could use a patched nerd font. This will support much more glyphs commonly used in terminals:

https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts

See https://github.com/elisescu/tty-share/issues/15

Reply:
Yeah, I think changing to a patched nerdfont could be good.
Source Code Pro (This version, at least) supports at least the powerline glyphs, I think it might support the same set of glyphs as nerd font.